config/config.sample.php

```diff
@@ -1055,6 +1055,20 @@
 	'timeout' => 0.0,
 	'password' => '', // Optional, if not defined no password will be used.
 	'dbindex' => 0,   // Optional, if undefined SELECT will not run and will use Redis Server's default DB Index. Out of the box, every Redis instance supports 16 databases so `<dbIndex>` has to be set between 0 and 15.
+	 // Optional config option
+	 // In order to use connection_parameters php-redis extension >= 5.3.0 is required
+	 // In order to use SSL/TLS redis server >= 6.0 is required
+	 // In case, don't forget to prefix the host with tls:// like tls://localhost
+	 // In case, SSL/TLS data **must** be in stream section
```

Comment:
```suggestion
	 // In a single-server configuration the SSL/TLS data **must** be in the stream section
```

Is this what is meant?